### DISCUSSION TITLE
How access to script-extensions on Docker?

### DISCUSSION THREAD

**Discussion by mevgdev** [plausible/analytics#1868] [Self-Hosted Support] [answered]
I don't found information at https://plausible.io/docs/script-extensions, about how configurate it on docker, because when try to access for ex to:
localhost:8000/js/plausible.js  => work it
but
localhost:8000/js/script.manual.js => dont exist like to https://plausible.io/js/script.manual.js

please helpme!

**Comment by AxelTheGerman** [comment, marked as answer]
I had the same issue, `exclusions` and `hash` seem to be the only 2 working for me (even combined)

Then I upgraded my self hosted instance from 1.3 to 1.4.4 and now it seems to work - not sure if they were added later or there was some other issue
